src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs

```diff
@@ -810,7 +810,7 @@ internal void HandleAltSvc(IEnumerable<string> altSvcHeaderValues, TimeSpan? res
                         break;
                     }
 
-                    if (nextAuthority == null && value.AlpnProtocolName == "h3")
+                    if (nextAuthority == null && value != null && value.AlpnProtocolName == "h3")
```

Comment:
@scalablecory it seems like value could null ref here because the Parser we are using support multiple values, so `TryParseValue` might return true and `parsedValue` could be null if `altSvcHeaderValue` is null or empty:
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Net.Http/src/System/Net/Http/Headers/BaseHeaderParser.cs#L40

Please confirm this fix is correct.

Reply:
Looks correct to me. I'll add a test in a later PR.